.github/workflows/ci.yml

```
@@ -36,36 +32,41 @@ jobs:
           source .venv/bin/activate
           flake8 .
 
-      - name: 🧪 Pytest - Unit Tests
+      - name: Run pytest
         run: |
           source .venv/bin/activate
-          pytest --cov=src --cov-report=xml tests/
-      - name: Upload coverage report
-        uses: codecov/codecov-action@v3
+          pytest --cov=src
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)